src/nix/profile.cc

```diff
@@ -978,14 +976,16 @@ struct CmdProfile : NixMultiCommand
         : NixMultiCommand(
             "profile",
             {
-              {"install", []() { return make_ref<CmdProfileInstall>(); }},
+              {"add", []() { return make_ref<CmdProfileAdd>(); }},
               {"remove", []() { return make_ref<CmdProfileRemove>(); }},
               {"upgrade", []() { return make_ref<CmdProfileUpgrade>(); }},
               {"list", []() { return make_ref<CmdProfileList>(); }},
               {"diff-closures", []() { return make_ref<CmdProfileDiffClosures>(); }},
               {"history", []() { return make_ref<CmdProfileHistory>(); }},
               {"rollback", []() { return make_ref<CmdProfileRollback>(); }},
               {"wipe-history", []() { return make_ref<CmdProfileWipeHistory>(); }},
+              // 2025-04-05 Deprecated in favor of "add"
+              {"install", []() { return make_ref<CmdProfileAdd>(); }},
```

Comment:
It's better to do this as a proper alias (see `aliases` in `main.cc`), otherwise `nix profile install` shows up as its own manual page which is probably not what we want. However `aliases` currently only handles top-level commands I think.